medusa/tv/series.py

```diff
@@ -2131,19 +2155,31 @@ def to_json(self, detailed=True):
             data['config']['release']['whitelist'] = bw_list.whitelist
 
         if detailed:
-            episodes = self.get_all_episodes()
             data['size'] = self.size
-            data['seasons'] = [list(v) for _, v in
-                               groupby([ep.to_json() for ep in episodes], lambda item: item['season'])]
+            data['showQueueStatus'] = self.show_queue_status
+            data['xemNumbering'] = [{'source': {'season': src[0], 'episode': src[1]},
```

Comment:
Since you're doing the same thing twice now (`xemNumbering` and `sceneNumbering` keys),
wouldn't it be better to have a helper function that returns it in the structure you want, simplifying `to_json()` a bit?